zingolib/src/blaze/block_witness_data.rs

```diff
@@ -603,18 +603,12 @@ impl BlockAndWitnessData {
 }
 
 fn is_orchard_tree_verified(determined_orchard_tree: String, unverified_tree: TreeState) -> bool {
-    if determined_orchard_tree == ORCHARD_START
+    determined_orchard_tree == ORCHARD_START
```

Comment:
What's a `determined_orchard_tree`?

Reply:
I suspect this term is a holdover from checkpoints...where a determined_tree would be one that we determined ourselves, rather than a pre-verified tree (one we loaded from a checkpoint)